napari/_qt/layer_controls/qt_vectors_controls.py

```diff
@@ -71,6 +76,23 @@ def __init__(self, layer) -> None:
         self.edge_color_label = QLabel(trans._('edge color:'))
         self._on_edge_color_change()
 
+        # dropdown to select the edge display vector_style
+        vector_style_comboBox = QComboBox(self)
+        for index, (data, text) in enumerate(VECTORSTYLE_TRANSLATIONS.items()):
+            data = data.value
+            vector_style_comboBox.addItem(text, data)
+            if data == self.layer.vector_style:
+                vector_style_comboBox.setCurrentIndex(index)
+
+        self.vector_style_comboBox = vector_style_comboBox
+        self.vector_style_comboBox.currentTextChanged.connect(
+            self.change_vector_style
+        )
+
+        # I had added the line below to mimic what is done with
+        # color mode, but it leads to a crash. Is it necessary?
+        # self._on_vector_style_change()
```

Comment:
```suggestion
```

Not necessary, since you already initialized it correctly with `setCurrentIndex` above.